### ISSUE TITLE
Uncaught ReferenceError: adTestentListener is not defined

### ISSUE THREAD

**Issue by WorkMAE** [austinksmith/Hamsters.js#53]
### New Issue Report

* ***What version of the library are you using?*** 5.1
* ***What operating system are you seeing the issue on?*** Windows 7
* ***Which platform (and or) browser are you seeing the issue on?*** Chrome
* ***What is the version of the platform (and or) browser you are seeing the issue on?*** Chrome 63/Angular JS 1.50

* ***Example logic***

I am trying to pass in Uint8Arrays and it hamsters.run and then gives me that error...

It appears I might be missing something? I have referenced the hamsters.js library properly as I set the hamsters.init and get it to run...
```

 hamsters.init({
                    browser: true,
                    reactNative: false,
                    node: false,
                    maxThreads: 16,
                    debug: 'verbose'
                });
```
and here is where I run hamsters:

```
var data = [];
                var myProm;
                for (var i = 0; i < 4; i++) {
                    let num = i + 1;
                    if (i === 3) {
                        user.readBinary(CVData.path + num + '.xlsb').then(function (response) {
                            data.push(new Uint8Array(response));
                        });
                    } else {
                        myProm = user.readBinary(CVData.path + num + '.xlsb').then(function (response) {
                            data.push(new Uint8Array(response));
                        });
                    }

                }
                //wait for the final promise to resolve

                myProm.then(function () {
                    var params = {
                        array: data,
                        aggregate: true
                    };

                    hamsters.run(params, function () {
                        params.array.forEach(function (item) {
                            var workbook = XLSX.read(item, { type: "array" });
                            var myJSON = XLSX.utils.sheet_to_json(workbook.Sheets.Sheet1);
                            rtn.data.push(myJSON);
                        });
                    }, function (results) {
                        console.log(results);
                    });
                });
    
```

Basically I want to pass in Uint8Arrays I am converting from ArrayBuffers I am getting from binary files I am loading in, use hamsters to do the processing work to turn it into a usable JSON object and then aggregate thees JSON objects into a single object with the results for each of the files.  Is this possible and if so, what am I doing wrong?

**Comment by austinksmith** [comment]
I cannot guarantee this will work but give something like this a shot, if you want I can debug this for you but it would have to be on a premium support paid option thing, and I would have to have a better understanding of your specific problem and what your data looks like before invoking the library.     



    var data = [];
                var myProm;
                for (var i = 0; i < 4; i++) {
                    let num = i + 1;
                    if (i === 3) {
                        user.readBinary(CVData.path + num + '.xlsb').then(function (response) {
                            data.push(new Array(response));
                        });
                    } else {
                        myProm = user.readBinary(CVData.path + num + '.xlsb').then(function (response) {
                            data.push(new Array(response));
                        });
                    }

                }
                //wait for the final promise to resolve

                myProm.then(function () {
                    var params = {
                        threads: data.length,
                        array: data,
                        aggregate: true,
                        dataType: 'UInt8'
                    };

                    hamsters.run(params, function () {
                        params.array.forEach(function (item) {
                            var workbook = XLSX.read(item, { type: "array" });
                            var myJSON = XLSX.utils.sheet_to_json(workbook.Sheets.Sheet1);
                            rtn.data.push(myJSON);
                        });
                    }, function (results) {
                        console.log(results);
                    });
                });
